lib/backends/webgl/ops/reshape.ts

```diff
@@ -1,45 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license.
 
-import {Attribute} from '../../../attribute';
 import {Reshape} from '../../../ops/reshape';
 import {Tensor} from '../../../tensor';
 import {ShapeUtil} from '../../../util';
-import {FunctionType, GlslPositionalFunction} from '../glsl-definitions';
-import {ShapeUtilsGlslLib} from '../glsl-shape-utils-lib';
 import {WebGLInferenceHandler} from '../inference-handler';
 import {getPackedShape} from '../utils';
 
 export class WebGLReshape extends Reshape {
-  initialize(attributes: Attribute): void {
-    this.outputShape = attributes.getInts('output_shape', []);
-    this.dimsToKeep = attributes.getInts('dims_to_keep', []);
-  }
-  getOutputShape(inferenceHandler: WebGLInferenceHandler, inputShapes: Array<ReadonlyArray<number>>):
-      ReadonlyArray<number> {
-    if (inputShapes.length >= 2) {
-      return inputShapes[1];
-    }
-    if (this.outputShape.length > 0) {
-      return this.outputShape;
-    }
-    if (this.dimsToKeep[0] === 0) {
-      return inputShapes[0]
-          .slice(0, this.dimsToKeep.length)
-          .concat(inputShapes[0].slice(this.dimsToKeep[this.dimsToKeep.length - 1] + 1).reduce((a, b) => a * b));
-    } else {
-      return [inputShapes[0].slice(0, this.dimsToKeep[0]).reduce((a, b) => a * b)].concat(
-          inputShapes[0].slice(this.dimsToKeep[0]));
-    }
-  }
   run(inferenceHandler: WebGLInferenceHandler, inputs: Tensor[]): Tensor[] {
-    const inputShape: ReadonlyArray<number> = inputs[0].dims.slice();
-    const inputShapes = [inputShape];
-    if (inputs.length > 1) {
-      inputShapes.push(ShapeUtil.calculateReshapedDims(inputShape, inputs[1].integerData));
-    }
+    const reshapedDims = ShapeUtil.calculateReshapedDims(inputs[0].dims, inputs[1].integerData);
```

Comment:
@fs-eire and @jeffsaremi - While trying to understand the reshape op in webgl - I saw some redundant code (I think!). Are you okay with this cleanup ?

Reply:
this looks good to me. thanks